scripts/run_benchmarking

```diff
@@ -5,28 +5,35 @@ set -x
 # Clean results folder
 mkdir -p /data/results
 mkdir -p /data/out
-rm /data/results/*
-rm /data/out/*
+rm /data/results/* || echo "No results"
+rm /data/out/* || echo "No output"
 
 # Java tests
 mvn -P metadata -Dtest.iterations=10 -Dtest.input=/data/BBBC/NIRHTa-001.ome.tiff -Dtest.output=/data/out/bbbc-java.ome.xml -Dtest.results=/data/results/bbbc-metadata-linux-java.tsv exec:java
 mvn -P metadata -Dtest.iterations=10 -Dtest.input=/data/mitocheck/00001_01.ome.tiff -Dtest.output=/data/out/mitocheck-java.ome.xml -Dtest.results=/data/results/mitocheck-metadata-linux-java.tsv exec:java
+mvn -P metadata -Dtest.iterations=10 -Dtest.input=/data/tubhiswt-4D/tubhiswt_C0_TP0.ome.tif -Dtest.output=/data/out/tubhiswt-java.ome.xml -Dtest.results=/data/results/tubhiswt-metadata-linux-java.tsv exec:java
 
 mvn -P pixels -Dtest.iterations=3 -Dtest.input=/data/BBBC/NIRHTa-001.ome.tiff -Dtest.output=/data/out/bbbc-java.ome.tiff -Dtest.results=/data/results/bbbc-pixeldata-linux-java.tsv exec:java
 mvn -P pixels -Dtest.iterations=3 -Dtest.input=/data/mitocheck/00001_01.ome.tiff -Dtest.output=/data/out/mitocheck-java.ome.tiff -Dtest.results=/data/results/mitocheck-pixeldata-linux-java.tsv exec:java
+mvn -P pixels -Dtest.iterations=3 -Dtest.input=/data/tubhiswt-4D/tubhiswt_C0_TP0.ome.tif -Dtest.output=/data/out/tubhiswt-java.ome.tiff -Dtest.results=/data/results/tubhiswt-pixeldata-linux-java.tsv exec:java
 
 # JACE tests
 cd $BF_JACE_HOME
 /install/bin/metadata-performance-jace 10 /data/BBBC/NIRHTa-001.ome.tiff /data/out/bbbc-jace.ome.xml /data/results/bbbc-metadata-linux-jace.tsv
 /install/bin/metadata-performance-jace 10 /data/mitocheck/00001_01.ome.tiff /data/out/mitocheck-jace.ome.xml /data/results/mitocheck-metadata-linux-jace.tsv
+# Throws boost: mutex exception
```

Comment:
Can't we just run this with `|| true` to avoid the abort?  We then get the statistics out of it--the error is only at program termination, not when running the tests.